src/h5web/dataset-visualizer/VisSelector.tsx

```diff
@@ -16,16 +16,15 @@ function VisSelector(props: Props): JSX.Element {
     <div className={styles.selector} role="tablist" aria-label="Visualization">
       {choices.map((vis) => {
         const { Icon } = VIS_DEFS[vis];
+        const onClick = onChange ? () => onChange(vis) : undefined;
```

Comment:
Doesn't matter, but you might be able to inline it nicely with ` &&` maybe:

```
onClick={ onChange && () => {
  onChange(vis);
}}
```

Reply:
I tried but TS didn't let me :cry:

Reply:
oh maybe it's `onChange?`, does it work with `| undefined` instead ? Again, no biggy either way.